media-libs/rtaudio/rtaudio-5.2.0.recipe 

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A set of C++ classes that provide a common API for realtime audio input/output"
+DESCRIPTION="This distribution of RtAudio contains the following:
+
+doc: RtAudio documentation (see doc/html/index.html)\
```

Comment:
this should not have \at the end of the lines and most likely be a list (with - or * at the start of the lines). This will be rendered in a non-standard, somewhat markdown-like way by HaikuDepot.